e2e/cypress/tests/integration/system_console/workspace_deletion_spec.js

```diff
@@ -10,7 +10,7 @@
 // Stage: @prod
 // Group: @system_console
 
-describe('Workspace deletion', () => {
+xdescribe('Workspace deletion', () => {
```

Comment:
Honestly, I won't recommend doing this since it will end up not executing to any group of tests (prod, unstable, etc) and could easily get forgotten. How about just removing the `@prod` tag so it will run in `unstable` group and then promote to `@prod` once the changes come in and the test run is known to be stable?

Reply:
sure @saturninoabril , I should have remove the tag. Thank you for suggesting this.